docs/schema.md

```diff
@@ -840,6 +851,7 @@ The defined ecosystems are:
 | `Bioconductor` | The biological R package ecosystem. The `name` is an R package name. |
 | `Bitnami` | Bitnami package ecosystem; the `name` is the name of the affected component. |
 | `Chainguard` | The Chainguard package ecosystem; the `name` is the name of the package. |
+| `CleanStart` | The CleanStart package ecosystem; the `name` is the name of the package. |
```

Comment:
Rather than placing it here, can you update the ecosystem.json, and run `scripts/update-ecosystem-lists.py` to generate the right changes.